### DISCUSSION TITLE
Does it support setting  download thread affinity?

### DISCUSSION THREAD

**Discussion by maoxing** [curl/curl#12686] [Q&A] [answered]
I will download a lot of resources, but I need to let the download thread started by curl run on the small core. Is there any relevant interface support now?

**Comment by bagder** [comment, marked as answer]
curl does not start any download threads by itself, it downloads in the thread you start it in.

**Reply by maoxing** [reply]
when using the multi interface to download resources at the same time, curl will not create any threads internally, right?

**Reply by bagder** [reply]
It might start threads for name resolving, but it never downloads anything in any other thread than in the one where it was started.

**Reply by maoxing** [reply]
got it, thanks!
